### PR TITLE
Create API contract for ML features

### PR DESCRIPTION
**Pull request by putuwaw**
I have created predict endpoint for ML features especially for sellers to do predicting food category, rating, and price.

### CODE REVIEW

File: paths/predict.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        content:
+          application/json:
+            schema:
+              $ref: "../components/schemas.yaml#/components/schemas/Response"
+            example:
+              status: true
+              code: 200
+              message: success get predict
+              data: {
+                "category": "donut",
```
Comment (tudemaha) [inline paths/predict.yaml]: Please change the type of category to integer. Use enum based on the list of categories provided by ML.

File: openapi.yaml [hunk 2]
```diff
@@ -44,4 +44,7 @@ paths:
   # report endpoint
   /reports:
     $ref: "./paths/report.yaml#/Report"
-    
\ No newline at end of file
+  
+  # ml seller endpoint
+  /predict:
```
Comment (tudemaha) [inline openapi.yaml]: Non mandatory. You can use any endpoint you want because the server for the ML is different with the backend service.

**Review by tudemaha — CHANGES_REQUESTED** [review]
Please note the review I have been added.

File: paths/predict.yaml (hunk 1 above)
Comment (putuwaw) [reply paths/predict.yaml]: Done, already changed it

File: openapi.yaml (hunk 2 above)
Comment (putuwaw) [reply openapi.yaml]: Aight, I've already remove it. But, in the FastAPI services, the endpoint is still same at `/predict`.

**Pull request merged**